portfolio/src/main/java/com/google/sps/servlets/ChartServlet.java

```diff
@@ -0,0 +1,42 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.LinkedHashMap;
+import java.util.Scanner;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that gets data from a csv and parses it as to use it in a chart.*/
+@WebServlet("/chart-data")
+public class ChartServlet extends HttpServlet {
+  private LinkedHashMap<String, Integer> memeSearchTrend = new LinkedHashMap<>();
+
+  /** Parses csv data a single time instead of on every request.*/
```

Comment:
Can you add the expectation that the CSV should be in the format '<date>, <interest value>'? And what the format of those specifically should be? (e.g. 'yyyy-mm-dd' and 'Integer', for instance)

Reply:
Done!